README.md

```diff
@@ -1,133 +1,55 @@
-
-If you are interested in mainaining the project, feel free to fork the repo, and if the fork is maintained I will gladly delegate you this repository or link it.
+The original author stopped using helm (which he mostly written this package for) on a daily basis, so I don't really keep up with the development of tree sitter, and nvim-tree-sitter.
+Some people have been found to keep the repo going, so if you have ideas for improvements feel free to open issues or pull requests.
 
 # tree-sitter-go-template
 
 [![Build/test](https://github.com/ngalaiko/tree-sitter-go-template/actions/workflows/ci.yaml/badge.svg)](https://github.com/ngalaiko/tree-sitter-go-template/actions/workflows/ci.yaml)
 
 [Golang templates][] grammar for [tree-sitter][].
 
-[tree-sitter]: https://github.com/tree-sitter/tree-sitter
-[Golang templates]: https://golang.org/pkg/text/template/
+Also includes a [helm][] dialect. The concept of a dialect is that it uses the same parser, but can have different queries (e.g. for different injections).
+More dialects could be added in the future (e.g. for html templates).
 
 ## NeoVim integration using [nvim-treesitter][]
 
-* Add gotmpl parser following [nvim-treesitter instructions][]
-  ```lua
-  local parser_config = require'nvim-treesitter.parsers'.get_parser_configs()
-  parser_config.gotmpl = {
-    install_info = {
-      url = "https://github.com/ngalaiko/tree-sitter-go-template",
-      files = {"src/parser.c"}
-    },
-    filetype = "gotmpl",
-    used_by = {"gohtmltmpl", "gotexttmpl", "gotmpl", "yaml"}
-  }
-  ```
-  Note that `yaml` is listed under `used_by`. I've set this to highlight [helm][] templates as Go Templates instead of yaml.
-  To ensure that yaml highlighting is still working, you should set up [language injection][] for gotmpl filetypes.
-
-* Run `:TSInstallFromGrammar gotmpl` to download and compile the grammar into your tree-sitter installation
-* Setup filetype detection in `~/.config/nvim/ftdetect/gotmpl.vim`:
-  ```vimscript
-  autocmd BufNewFile,BufRead * if search('{{.\+}}', 'nw') | setlocal filetype=gotmpl | endif
-  ```
-* Define language injection for yaml in ~/.config/nvim/queries/gotmpl/injections.scm:
-  ```scheme
-  (text) @yaml
-  ```
-* Define highlights in `~/.config/nvim/queries/gotmpl/highlights.scm`, for example:
-  ```scheme
-  ; Identifiers
-
-  [
-      (field)
-      (field_identifier)
-  ] @property
-
-  (variable) @variable
-
-  ; Function calls
-
-  (function_call
-    function: (identifier) @function)
-
-  (method_call
-    method: (selector_expression
-      field: (field_identifier) @method))
-
-  ; Operators
-
-  "|" @operator
-  ":=" @operator
+-   Setup filetype detection:
 
-  ; Builtin functions
+    -   for [helm][] the [vim-helm] plugin is recommended
```

Comment:
I don't think recommending this plugin is a good idea. Users would only use it for file type detection, since highlighting is done by tree-sitter.